### PR TITLE
Remove deprecated method

### LINKED ISSUE #15870: Remove Test.getClassLoaderCache()
### Summary

- Test.getClassLoaderCache()

### Checklist
- [ ] Validate whether we should remove the API in the 7.0 release
  - If the removal is not possible then create a follow-up issue and link it in the [overview spreadsheet](https://docs.google.com/spreadsheets/d/19J1nR_dFKpfKdu5KDFMVZGfjR0ysT9DthsBUPwf8mkM/edit#gid=493905624) 
- [ ] Update the upgrade guide
- [ ] Check User Manual for dangling references and outdated documentation
- [ ] Mark the story as done in the [overview spreadsheet](https://docs.google.com/spreadsheets/d/19J1nR_dFKpfKdu5KDFMVZGfjR0ysT9DthsBUPwf8mkM/edit#gid=2040581133).

### PR DESCRIPTION
**Pull request by bmuskalla**
Fixes #15870

### CODE REVIEW

**Comment by bmuskalla** [comment]
@bot-gradle test QFL

**Comment by bmuskalla** [comment]
@bot-gradle test this

File: subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java [hunk 1]
```diff
@@ -1165,7 +1152,6 @@ void setTestExecuter(TestExecuter<JvmTestExecutionSpec> testExecuter) {
      *
      * @since 6.7
      */
-    @Incubating
```
Comment (jjohannes) [inline subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java:1168]: This method was already added to the list in the release notes :+1:

**Review by jjohannes — APPROVED** [review]

File: subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java (hunk 1 above)
Comment (bmuskalla) [reply subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java:1168]: Thanks

**Pull request merged**